File: source/includes/code/go/crud-update-final-query.go
package main

import (
	"context"
	"fmt"
	"log"
	"os"

	"go.mongodb.org/mongo-driver/bson"
	"go.mongodb.org/mongo-driver/mongo"
	"go.mongodb.org/mongo-driver/mongo/options"
)

func main() {
	uri := "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&writeConcern=majority"

	client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
	if err != nil {
		panic(err)
	}

	defer func() {
		if err = client.Disconnect(context.TODO()); err != nil {
			panic(err)
		}
	}()
	
	// database and colletion code goes here
	db := client.Database("sample_guides")
	coll := db.Collection("comets")

	// update code goes here
	filter := bson.D{{}}
	update := bson.D{{"$mul", bson.D{{"Radius", 1.60934}}}}

	result, err := coll.UpdateMany(context.TODO(), filter, update)
	if err != nil {
		panic(err)
	}

	// amount update code goes here
	fmt.Println(result.ModifiedCount)
}

File: source/includes/code/python/crud_update_final_query.py
from pymongo import MongoClient

# Replace the uri string with your MongoDB deployment's connection string.
uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&writeConcern=majority"

client = MongoClient(uri)

# database and collection code goes here
db = client.sample_guides
coll = db.comets

# update code goes here
doc = {"$mul": {"Radius": 1.60934}}
result = coll.update_many({}, doc)

# display the results of your operation
print("Number of documents updated: ", result.modified_count)

# Close the connection to MongoDB when you're done.
client.close()
